src/CachedModel.php

```diff
@@ -55,7 +55,7 @@ public static function all($columns = ['*'])
         $class = get_called_class();
         $instance = new $class;
         $tags = [str_slug(get_called_class())];
-        $key = 'genealabslaravelmodelcachingtestsfixturesauthor';
+        $key = sha1('genealabslaravelmodelcachingtestsfixturesauthor');
```

Comment:
Call `makeCacheKey()` here, this was bad code on my part.

Reply:
This won't work in a static function.

Reply:
I'm confused as to why the function is marked static in the first place. It is called like an object method.

Reply:
Something like this?
```php
$key = $instance->makeCacheKey();
```

Reply:
for instance in `tests/Unit/CachedBuilderTest.php` line 44.